src/OpenApi/OpenApiResponseBody.php

```diff
@@ -22,12 +22,19 @@ class OpenApiResponseBody extends Body
      */
     public function match($body)
     {
-        if (!isset($this->structure['content'])) {
+        if (!isset($this->structure['content']) && !isset($this->structure['$ref'])) {
             if (!empty($body)) {
                 throw new NotMatchedException("Expected empty body for " . $this->name);
             }
             return true;
         }
-        return $this->matchSchema($this->name, $this->structure['content'][key($this->structure['content'])]['schema'], $body);
+        
+        if(!isset($this->structure['content']) && isset($this->structure['$ref'])){
+            $defintion = $this->schema->getDefinition($this->structure['$ref']);
+            return $this->matchSchema($this->name, $defintion, $body);
+        }
+        else {
```

Comment:
Since the command inside the IF conditional equals true is a `return` it is not necessary to have the `else` statement.